client-common/src/main/kotlin/io/provenance/client/common/exceptions/TransactionTimeoutException.kt

```diff
@@ -0,0 +1,6 @@
+package io.provenance.client.common.exceptions
+
+import io.grpc.Status
+import io.grpc.StatusRuntimeException
+
+class TransactionTimeoutException(message: String): StatusRuntimeException(Status.DEADLINE_EXCEEDED.withDescription(message))
```

Comment:
I made this exception extend from a StatusRuntimeException to try and keep the behavior of catching a StatusRuntimeException from a failure in the old BROADCAST_MODE_BLOCK... interested in input here as to whether this is a good idea or not

Reply:
With the goal being to minimize hassle of adoption for clients who do not want to implement their own solution, I think this is a good solution.

Reply:
agreed